src/App.js

```diff
@@ -57,11 +57,13 @@ class App extends React.Component {
       time: getCurrentTime(),
       // Starts with welcome state, paste in whatever state you want to work on
```

Comment:
These could all be declared with the useState hook:

https://reactjs.org/docs/hooks-state.html

`[time, setTime] = useState(getCurrentTime())` 

and so forth